tests/integration/kubernetes/lib.sh

```diff
@@ -94,11 +97,41 @@ assert_pod_fail() {
 	! k8s_create_pod "$container_config" || /bin/false
 }
 
+
+# Check the pulled rootfs on host for given node and sandbox_id
+#
+# Parameters:
+#	$1 - the k8s worker node name
+#	$2 - the sandbox id for kata container
+#	$3 - the expected count of pulled rootfs
+#
+assert_rootfs_count() {
+	local node="$1"
+	local sandbox_id="$2"
```

Comment:
It must check that `sandbox_id` is non-empty otherwise the `exec_host $node "find /run/kata-containers/shared/sandboxes/${sandbox_id} -name rootfs -type d`  below may return an unexpected number of rootfs.